ballerina-tests/tests/37_interfaces.bal

```diff
@@ -94,3 +94,58 @@ isolated function testInterfacesWithInterfaceTypeArray() returns error? {
     json expectedPayload = check getJsonContentFromFile("interfaces_with_interface_type_array.json");
     assertJsonValuesWithOrder(actualPayload, expectedPayload);
 }
+
+@test:Config {
+    groups: ["interfaces", "fragments"]
+}
+isolated function testInterfaceImplementingInterfaceByQueryingOnInterface() returns error? {
+    string document = check getGraphQLDocumentFromFile("interfaces_implementing_interface_querying_on_interface.graphql");
+    string url = "http://localhost:9089/interfaces_implementing_interface";
+    json actualPayload = check getJsonPayloadFromService(url, document);
+    json expectedPayload = check getJsonContentFromFile("interfaces_implementing_interface_querying_on_interface.json");
+    assertJsonValuesWithOrder(actualPayload, expectedPayload);
+}
+
+@test:Config {
+    groups: ["interfaces", "fragments"]
+}
+isolated function testInterfaceImplementingInterfaceByQueryingOnTransitiveType() returns error? {
+    string document = check getGraphQLDocumentFromFile("interfaces_implementing_interface_querying_on_transitive_type.graphql");
+    string url = "http://localhost:9089/interfaces_implementing_interface";
+    json actualPayload = check getJsonPayloadFromService(url, document);
+    json expectedPayload = check getJsonContentFromFile("interfaces_implementing_interface_querying_on_transitive_type.json");
+    assertJsonValuesWithOrder(actualPayload, expectedPayload);
+}
+
+@test:Config {
+    groups: ["interfaces", "fragments"]
+}
+isolated function testInterfaceImplementingInterfaceByQueryingOnBothTransitiveTypeAndInterface() returns error? {
+    string document = check getGraphQLDocumentFromFile("interfaces_implementing_interface_querying_on_transitive_type_and_interface.graphql");
```

Comment:
Check the character limits on these lines. Seems like they exceed the limit.

IMHO, these function names are quite too long. Too long function names are kinda hard to read.

Reply:
Yes, this is due to the prefix `testInterfaceImplementingInterface`.
We could replace the suffixes of function name with number series ex:`testInterfaceImplementingInterface2` since the query file name describes the test. WDYT ?

Reply:
Usually, we named the query file similar to the test function name.

Reply:
Removed prefix and updated the query filenames